info.bibletime.BibleTime.json

```diff
@@ -0,0 +1,105 @@
+{
+  "id": "info.bibletime.BibleTime",
+  "sdk": "org.kde.Sdk",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "command": "bibletime",
+  "default-branch": "master",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--socket=wayland",
+    "--share=network",
+    "--persist=.bibletime",
+    "--persist=.sword",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "clucene",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=release"
+      ],
+      "builddir": true,
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://clucene.git.sourceforge.net/gitroot/clucene/clucene",
+          "commit": "e8e3d20f20da5ee3e37d347207b01890829a5475"
+        }
+      ],
+      "cleanup": [
+        "/lib/pkgconfig",
+        "/lib/CLuceneConfig.cmake",
+        "/include"
+      ]
+    },
+
+    {
+      "name" : "sword",
+      "config-opts" : [
+        "--with-icu=no --disable-static"
+      ],
+      "sources" : [
+        {
+          "url" : "http://crosswire.org/ftpmirror/pub/sword/source/v1.8/sword-1.8.1.tar.gz",
+          "sha256" : "ce9aa8f721a737f406115d35ff438bd07c829fce1605f0d6dcdabc4318bc5e93",
+          "type" : "archive"
+        }
+      ],
+      "cleanup": [
+        "/lib/pkgconfig",
+        "/lib/libsword.la",
+        "/include"
+      ]
+    },
+    {
+      "name": "bibletime-docs",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/bibletime/generated_docs.git"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+          "mkdir -p /app/share/doc/bibletime",
+          "cp -r docs/v3.0_rc1/h*  /app/share/doc/bibletime"
+      ]
+    },
+    {
+      "name": "bibletime",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=release","-DBT_RUNTIME_DOCDIR=share/doc/bibletime","-DBUILD_HANDBOOK_HTML=OFF","-DBUILD_HANDBOOK_PDF=OFF","-DBUILD_HOWTO_HTML=OFF","-DBUILD_HOWTO_PDF=OFF"
+      ],
+      "builddir": true,
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/bibletime/bibletime.git",
+          "branch": "stable-3.0",
+          "commit": "f6e1e67d3e19618c2bf3d58f722ca20f37374813"
+        }
+      ],
+      "cleanup": [
+        "/lib/pkgconfig"
+      ]
+    },
+    {
+      "name": "appdata",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p ${FLATPAK_DEST}/share/appdata",
+        "cp info.bibletime.BibleTime.appdata.xml ${FLATPAK_DEST}/share/appdata"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "info.bibletime.BibleTime.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html